docs/tutorial/handler.rst

```diff
@@ -126,13 +218,58 @@ for instantiating all the *Handlers* with the desired configuration
 and attaching them to the *Router* to process the matching
 *Message* type (identified by it's *Path*):
 
-**TODO** include our code
+.. literalinclude:: ../../examples/tutorial/x/blog/handlers.go
+    :language: go
+    :lines: 21-26
+
+Testing Handlers
```

Comment:
Thank you for adding this section. It definitely gives some good starting points for people to figure out how to write tests, what utils are available.